GeorgianMemenator/Scenes/DogBreedDetail/DogBreedDetailViewModel.swift

```diff
@@ -24,11 +27,15 @@ class DogBreedDetailViewModel: ObservableObject {
             .getDogBreedDetails(with: dogBreedId)
```

Comment:
Before you call service function to fetch details, you need to set `isLoading = true`, so that view understand that it should show loading.

Reply:
I fixed everything ✅